lib/textrazor/api_client.rb

```diff
@@ -0,0 +1,36 @@
+require 'rest_client'
```

Comment:
What's the main reason to separate this out into an `api_client` class? `

Reply:
Agreed. I'll rework this part.